logstash-filter-tld.gemspec

```
@@ -17,14 +17,12 @@ Gem::Specification.new do |s|
   # Special flag to let us know this is actually a logstash plugin
   s.metadata = { "logstash_plugin" => "true", "logstash_group" => "filter" }
 
-  # public_suffix 3.x+ includes ruby syntax from 2.1
-  # This effectively requires Logstash >= 6.x
-  s.required_ruby_version = '>= 2.1.0'
+  # public_suffix 5.x+ includes ruby syntax from 2.6
+  s.required_ruby_version = '>= 2.3'
 
   # Gem dependencies
   s.add_runtime_dependency "logstash-core-plugin-api", ">= 1.60", "<= 2.99"
-  s.add_runtime_dependency 'public_suffix', '~>3'
+  s.add_runtime_dependency 'public_suffix', '~> 4'
```

So we allow 4.x and 5.x:

```suggestion
  s.add_runtime_dependency 'public_suffix', '> 3', '< 6'
```